### DISCUSSION TITLE
ID , Linking missing in label.txt

### DISCUSSION THREAD

**Discussion by SherwinSamuel23** [PaddlePaddle/PaddleOCR#12720] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 10
- 版本号/Version：Paddle：2.5.1  PaddleOCR： 2.7.0.3问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：PPOCRLabel = 2.1.3
After labeling image in label.txt id and linking are missing  

dataset/page_1.jpg	[{"transcription": "iicici Bank", "points": [[360, 409], [778, 409], [778, 485], [360, 485]], "difficult": false, "key_cls": "Bank"}, {"transcription": "IFS CodeICIC0000395", "points": [[546, 549], [826, 549], [826, 583], [546, 583]], "difficult": false, "key_cls": "IFSC"}, {"transcription": "AlcNo758205201489", "points": [[268, 853], [800, 853], [800, 903], [268, 903]], "difficult": false, "key_cls": "AC"}]

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by MirzaAhmedBaig** [comment]
yes

**Comment by SWHL** [comment]
Since PPOCRLabel has moved, please go to [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel) to ask questions if you have any questions!
This issue has been closed first.
